src/reducers/book.js

```diff
@@ -1,9 +1,17 @@
+import faker from 'faker';
 import * as types from '../actions/actionTypes';
 
 function bookReducer(state = [], action) {
   switch (action.type) {
     case types.CREATE_BOOK:
-      return [...state, action.payload];
+      return [
+        ...state,
+        {
+          id: faker.finance.routingNumber(),
```

Comment:
- [ ] [OPTIONAL] Consider using the uuid package here for unique ids generation or implementing your own id incrementer in the store.